### PR TITLE
Tweaks to the credential and connection sections

### PR DESCRIPTION
**Pull request by lamagnifica**
@stroilova, please let me know of any other changes that might be
helpful

### CODE REVIEW

**Comment by lamagnifica** [comment]
@mulby , would you please take a look at this commit? https://github.com/edx/edx-documentation/commit/0737d3b203a73c35ea6c5773208182f36ade311a
It adds info about stop_video from TNL-2008, https://github.com/edx/edx-platform/pull/7797

**Comment by lamagnifica** [comment]
@BenjiLee would you please review the changes to the en_us/data/source/internal_data_formats/tracking_logs.rst file from a mobile perspective? 
(triggered by TNL-2008, edx/edx-platform#7797)

**Comment by BenjiLee** [comment]
Looks like edx/edx-platform#7797 fixes a bug where the `stop_video` event was not being emitted.  This doc change looks like we are emitting both a `pause_video` and `stop_video`.

**Comment by lamagnifica** [comment]
@mulby @BenjiLee please review en_us/data/source/internal_data_formats/tracking_logs.rst for accuracy

File: en_us/data/source/internal_data_formats/tracking_logs.rst [hunk 1]
```diff
@@ -818,10 +818,22 @@ for the :ref:`play_video` events.
 *************************************
 
 When a user selects the video player's **pause** control, the player emits a
-``pause_video`` event. The player also emits this event when it reaches the
-end of the video file and play automatically stops. Note that as of June 2014,
-the player also emits a ``stop_video`` event when it reaches the end of the
-video file and play automatically stops.
+``pause_video`` event. For videos that are streamed in a browser, the player
+also emits this event when it reaches the end of the video file and play
+automatically stops, in addition to the ``stop_video`` event.
```
Comment (mulby) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: This is actually _always_ emitted at the end of the video (in browser and mobile).

File: en_us/data/source/internal_data_formats/tracking_logs.rst [hunk 2]
```diff
@@ -818,10 +818,22 @@ for the :ref:`play_video` events.
...
+
+Note that course teams can specify a **Video Stop Time** for video files. 
+
+* If the user streams a video file in a browser and a **Video Stop Time** is
+  present for the video, the player stops at the specified time and emits the
+  ``pause_video`` and ``stop_video`` events.
+
+* If the user downloads a video file to view with the edX mobile app, the
+  player ignores the **Video Stop Time** and plays the file to its end. The
+  player then emits only the ``stop_video`` event.
```
Comment (mulby) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: It actually emits a stop_video and a pause_video event at the end of all videos.

Comment (lamagnifica) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: @BenjiLee , I'd appreciate your take on this

Comment (BenjiLee) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: @mulby We have not implemented this in the mobile app. This is the open ticket. https://openedx.atlassian.net/browse/MA-338. Last I checked, only a stop event is emitted.

File: en_us/data/source/internal_data_formats/tracking_logs.rst (hunk 1 above)
Comment (lamagnifica) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: @BenjiLee , @mulby 's comment here doesn't agree with the info I got from you earlier today.
@mulby, even newer info than in this commit: the mobile app does not emit pause_video events when the file autostops. This is true whether a downloaded file is being played by the app, or the file is being streamed in the app.
See https://openedx.atlassian.net/browse/MA-338

Comment (mulby) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: Ah, I thought they had already done this.

**Comment by lamagnifica** [comment]
@mulby @BenjiLee ?

File: en_us/data/source/internal_data_formats/tracking_logs.rst (hunk 1 above)
Comment (BenjiLee) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: We have not.

**Comment by BenjiLee** [comment]
:+1:

**Comment by mulby** [comment]
+1

**Comment by lamagnifica** [comment]
@BenjiLee , @mulby , could this PR be causing the confusion about whether we have or have not fixed when stop_ is emitted? https://github.com/edx/edx-platform/pull/7797/files
(I am completely confused by this at this point)

**Comment by mulby** [comment]
It is likely adding to the confusion.

The current situation is (AFAIK):
- Stop video is emitted at the end of a video for both mobile and web video players
- Stop video is emitted at the end of the studio configured video segment only on the web player
- Pause video is emitted at the end of the video and at the end of the configured video segment only by the web player
- Pause video is not emitted by the mobile video player unless the user presses the "pause" button, so it is not emitted at the end of the video and not emitted at the end of a configured video segment

File: en_us/data/source/internal_data_formats/tracking_logs.rst (hunk 1 above)
Comment (lamagnifica) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: @BenjiLee , please weigh in whether https://github.com/edx/edx-platform/pull/7797 means that I can revert this text back to what it was previously. I think that it only affects stop_video event,s but I'm not entirely clear on what all is happening with these...

Comment (BenjiLee) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: There should be no change to the doc for the stop event.  edx/edx-platform#7797 addresses a bug where a stop event which should have been emitting, was not emitting.

**Comment by BenjiLee** [comment]
@mulby This is also my understanding.

**Pull request merged**